Mlem/Views/Shared/Links/User Profile Label.swift

```diff
@@ -62,15 +62,15 @@ struct UserProfileLabel: View {
                         Image(systemName: "person.circle")
                             .resizable()
                             .scaledToFill()
-                            .frame(width: AppConstants.largeAvatarSize, height: AppConstants.defaultAvatarSize)
+                            .frame(width: AppConstants.largeAvatarSize, height: AppConstants.largeAvatarSize)
                             .foregroundColor(.secondary)
                     }
                 }
             } else {
                 Image(systemName: "person.circle")
                     .resizable()
                     .scaledToFill()
-                    .frame(width: AppConstants.largeAvatarSize, height: AppConstants.defaultAvatarSize)
+                    .frame(width: AppConstants.largeAvatarSize, height: AppConstants.largeAvatarSize)
```

Comment:
When we duplicate the same SwiftUI view across multiple parts of the app, it might be a good idea to make it a component, at least internal to this view.... for readability.

Reply:
I'll slate it for 1.1